src/fragments/lib/auth/js/set-up-auth.mdx

```diff
@@ -476,7 +475,7 @@ The `amplify-authenticator` should wrap sections of your app that you want to pr
 </BlockSwitcher>
 
 Once you add the Authenticator component to your app, you can test the sign-up, sign-in, and sign-out functionality. You can also customize the Authenticator UI component to adjust colors and styling as needed.
-
+ 
```

Comment:
Do we want to add a link to the UI Library - is there a method to link with a switcher or can we link to a default like React? https://ui.docs.amplify.aws/react/connected-components/authenticator/customization

Reply:
yes, that sounds good!